ruby/lib/libdatadog/version.rb

```diff
@@ -2,11 +2,11 @@
 
 module Libdatadog
   # Current libdatadog version
-  LIB_VERSION = "11.0.0"
+  LIB_VERSION = '12.0.0'
```

Comment:
The gem is using standardrb as well, which defaults strings to `"`.

This will break once we try to run `docker-compose run push_to_rubygems` since it runs the auto-formatter first which will undo this and then complain "There are files that need to be committed first."

```suggestion
  LIB_VERSION = "12.0.0"
```

Reply:
Uh weird, this was autoformatted. Let me check why the autoformat was not using standard